dames.c

```diff
@@ -1,6 +1,246 @@
 #include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <stdbool.h>
 
-int dummy_fun(void){
-	printf("Hello yall\n");
-	return 0;
+#define GRIDSIZE 10
+
+#define WHITE_CHECKER 1
+#define WHITE_DAME 2
```

Comment:
It's probably a bad practice to add symbols that won't be used, it's slowing down the review.